.changeset/large-baboons-hope.md

```diff
@@ -0,0 +1,5 @@
+---
+'@clerk/clerk-js': patch
+---
+
+Added some padding when the ArrowBlockButton is in loading state and has a left icon to prevent layout shift.
```

Comment:
Nitpicky comment follows :)

The text here will be included in our changelog with the next production release and it's intended to be shown to Clerk customers. So I would rephrase it as such:

```suggestion
Updated the OAuth buttons in the SignIn and SignUp components to prevent layout shifts while loading.
```